ITE/9/README.adoc

```diff
@@ -45,7 +45,32 @@ The link:https://github.com/in-toto/attestation/[in-toto Attestation Framework]
 
 When discussing a new attestation type, what is really being defined is a new type of link:https://github.com/in-toto/attestation/tree/main/spec#predicate[predicate]. The developer(s) of this new predicate must create a formal document that specifies the predicate type, and discusses several other related aspects. The purpose and prerequisites for the new attestation type MUST be defined. The document should also include a model that describes how the various components of the predicate map to supply chain semantics, juxtaposed with detailed use cases. These use cases SHOULD include example metadata that clearly highlight how the proposed predicate addresses the shortcoming described by them. Including these aspects will aid readers and future adopters understand the why and how of a particular attestation type.
 
-The specification MUST contain a detailed schema of the fields that make up the predicate. This schema MUST define unambiguously a unique `predicateType` following the rules defined by the attestation framework. The schema must also clearly define the types of every field, with ranges for values where applicable. Further, the specification MUST contain rules for parsing the fields, as well as make recommendations for handling edge cases, where parsing or handling of fields would deviate from the standard link:https://github.com/in-toto/attestation/tree/main/spec/v1.0-draft#parsing-rules[parsing rules]. For example, it may be preferable in some cases to silently allow extra, undefined fields, while in other cases such metadata should be rejected.
+The specification's front matter MUST unambiguously declare a predicate type, a
+version, and a shorter predicate name following the rules defined by the
```

Comment:
Do these need to be unique?  Is there a registry of these where all the used names are listed?

Reply:
Yes, they need to be unique, good catch. We don't _define_ a formal registry but note that ITE-9 is for vetting predicates and listing them here: https://github.com/in-toto/attestation/tree/main/spec/predicates.

So, the predicate name has to be unique to not collide with an existing file. I'll make that explicit in ITE-9. The predicate type is already required to be unique in the attestation spec but I'll also make it explicit here.

Reply:
Done!